File: Python/reservation.py
from datetime import date
from objet_louable import ObjetLouable

class Reservation:
    def __init__(self, objet: ObjetLouable, date: date):
        self._objet = objet
        self._date = date

    @property
    def objet(self):
        return self._objet

    @property
    def date(self):
        return self._date

File: Js/Reservation.js
const ObjetLouable = require('./objetLouable');

class Reservation {
    #objet;
    #date;

    constructor(objet, date) {
        this.#objet = objet;
        this.#date = date;
    }

    get objet() {
        return this.#objet;
    }

    get date() {
        return this.#date;
    }
}

module.exports = Reservation;